Plugins/NativeXr/Include/Babylon/Plugins/NativeXr.h

```diff
@@ -2,7 +2,24 @@
 
 #include <napi/env.h>
 
-namespace Babylon::Plugins::NativeXr
+namespace Babylon::Plugins
 {
-    void Initialize(Napi::Env env);
+    class NativeXr final
+    {
+    public:
+        class Impl;
+
+        NativeXr(std::shared_ptr<Impl> impl);
```

Comment:
Move to private?

Reply:
`Impl` is public because it is also used by the Napi JS wrappers, but I moved the constructor.